package.json

```
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@patternfly/patternfly": "^2.23.0",
+    "@patternfly/react-core": "^3.58.1",
+    "@patternfly/react-icons": "^3.10.13",
```

Is it better to fix the version of this?

```suggestion
"@patternfly/react-icons": "3.10.13"
```